scripts/update_docs.sh

```diff
@@ -0,0 +1,76 @@
+#!/bin/sh
+
+set -x
+
+SCRIPTS_DIR="$( cd "$( dirname "$0" )" >/dev/null 2>&1 && pwd )"
+DIR=$(cd $SCRIPTS_DIR; cd ..; pwd)
+VERSION=$1
+TOC_NAME=$(echo ${VERSION}toc|sed s/\\./-/g)
+
+read -r -d '' CONFIG_VERSION_BLOCK << EOM
```

Comment:
These `read` commands are failing locally for me. I'm getting the error: ``

Again, it seems to be a difference in `read` in `sh` vs `bash`. If I use `bash` in the shebang at the top of the script it works as expected. Can we use bash instead?

Reply:
Absolutely; no loyalty, just habit. Let me know if the new version works for you.

Reply:
It works :) thanks!